paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -643,6 +643,7 @@ class CustomerSheetViewModelTest {
             val resultTurbine = viewModel.result.testIn(backgroundScope)
 
             assertThat(viewStateTurbine.awaitItem()).isInstanceOf<SelectPaymentMethod>()
+            viewStateTurbine.cancelAndIgnoreRemainingEvents()
```

Comment:
This is required because turbine has a behavior change where they now assert on unconsumed events when the scope is cancelled by the outer scope.